lib/facter/resolvers/networking_linux.rb

```diff
@@ -240,7 +271,15 @@ def search_internal_leases(interface_name)
           return unless dhcp
 
           dhcp = dhcp.match(/SERVER_ADDRESS=(.*)/)
-          dhcp[1] if dhcp
+          interface_data[:dhcp] = dhcp[1] if dhcp
+        end
+
+        def search_dhcpcd_command(interface_data, interface_name)
```

Comment:
This methos should be named `search_with_dhcpcd_command` or `search_dhcp_with_dhcpcd`